modules/ocf_tv/manifests/pulse.pp

```diff
@@ -0,0 +1,10 @@
+class ocf_tv::pulse {
+  include ocf::packages::pulse;
+
+  # Make pulse listen on 127.0.0.1:4713 desktops can SSH tunnel to it
+  exec { 'enable-pulse-tcp-listen':
+    command => 'echo "load-module module-native-protocol-tcp auth-ip-acl=127.0.0.1 auth-anonymous=1" >> /etc/pulse/default.pa',
```

Comment:
I'd probably just puppet the entire default.pa file as opposed to doing this, seems cleaner

Reply:
Have you seen the entire file? I don't think we should commit that whole thing to add just one line. We already do this elswere. We already use this pattern in a few other places, e.g.
https://github.com/ocf/puppet/blob/master/modules/ocf_desktop/manifests/sshfs.pp#L20-L24
https://github.com/ocf/puppet/blob/master/modules/ocf_rancid/manifests/init.pp#L5-L20

Reply:
is there a way to use augeas for this? e.g. https://github.com/tahabi/puppet/blob/15b16e5f80c0dc1a165ab8d1abcaa4f4116ecdf9/modules/ocf_desktop/manifests/pulse.pp#L24

Reply:
Yep, here's what it looks like:
```puppet
$tcp_module = 'module-native-protocol-tcp auth-ip-acl=127.0.0.1 auth-anonymous=1'
augeas { 'enable-pulse-tcp':
  incl    => '/etc/pulse/default.pa',
  lens    => 'Spacevars.lns',
  changes => "set load-module[. = '${tcp_module}'] '${tcp_module}'",
}
```
It took me about an hour to learn how to write this and the only thing that makes it look cleaner is the fact that I assigned the setting to a variable. For one-liners like this, I think augeas hurts readability more than it helps.

Reply:
Lol, nice. Apologies for making you go through the effort.

I guess I'll update `ocf-tv` to reflect the changes made here and hopefully that results in a more secure but equally reliable way to tunnel audio to the TV.